www/cli/purgearchive.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/* Based on prunearchive.php (which relies on archive2.php being run)
+ * This takes a simpler approach: just delete the day folders in archive_dir,
+ * if they are older than a month.
+ */
+
+chdir('..');
+
+include 'common.inc';
+ignore_user_abort(true);
+set_time_limit(86400);   // only allow it to run for 1 day
+
+// bail if we are already running
+$lock = Lock("Remove Archive");

```

Comment:
nit: can you make the lock name "purgearchive" so it is clear that it is tied to this script?